pypeit/spectrographs/keck_kcwi.py

```diff
@@ -921,4 +929,44 @@ def get_datacube_bins(self, slitlength, minmax, num_wave):
         spec_bins = np.arange(1+num_wave) - 0.5
         return xbins, ybins, spec_bins
 
+    def flatfield_structure(self, ff_struct, gpmask):
+        r"""
+        Perform a model fit to the flatfield structure of KCWI data.
+        A few different setups were inspected (BH2 & BM with different
+        grating angles), and a very similar structure was found for all
+        setups, indicating that this structure is something to do with
+        the detector. The starting parameters and functional form are
+        assumed to be sufficient for all setups.
```

Comment:
According to Keck folks, this is some kind of transient feature. You may want to chat with them about that (Max) and also add that to the docs here.